docs/changelog/2024.md

```diff
@@ -71,43 +71,13 @@ These possibilities are now leveraged in all schema modules.
 - 🎨 Leverage a more concise way for defining registries: [lnschema-core](https://github.com/laminlabs/lnschema-core/pull/452) [findrefs](https://github.com/laminlabs/findrefs/pull/2) [ourprojects](https://github.com/laminlabs/ourprojects/pull/2) [clinicore](https://github.com/laminlabs/clinicore/pull/14) [cellregistry](https://github.com/laminlabs/cellregistry/pull/2) [@sunnyosun](https://github.com/sunnyosun)
 - 🎨 Squash migrations for faster instance creation: [lnschema-core](https://github.com/laminlabs/lnschema-core/pull/453) [bionty](https://github.com/laminlabs/bionty/pull/162) [wetlab](https://github.com/laminlabs/wetlab/pull/75) [@sunnyosun](https://github.com/sunnyosun)
 
-Bionty changes.
-
-- ✨ Add support for local source files in `sources.yaml` [PR](https://github.com/laminlabs/bionty/pull/156) [@mossjacob](https://github.com/mossjacob)
-
-It is now possible to add local parquet files as source using the `file://` prefix in the `sources.yaml` file.
-For example:
-
-```yaml
-Gene:
-ensembl:
-  arabidopsis thaliana:
-    release-57:
-      url: file:///path/to/local/df_arabidopsis thaliana__ensembl__release-57__Gene.parquet
-```
-
-Note that we discourage modifying the `sources.yaml` file because we are planning to deprecate it soon in favor of only modifying `Source` records.
-
-- ✨ Support `taxa="plants"` in `EnsemblGene` [PR](https://github.com/laminlabs/bionty/pull/153) [@mossjacob](https://github.com/mossjacob)
```

Comment:
`EnsembleGene` isn't user-facing, so I replaced it with `bionty.Gene` -- not sure if this is actually going to work

Reply:
No it isn't going to work.

Reply:
Being able to model plant genes is a fundamental new capability, I'd say, so I think this is awesome.

Reply:
To be more clear. This enables us to fetch plant genes from ensembl to create the parquet files and add them to bionty-assets. This is NOT user facing.

Reply:
But I thought there was a PR to bionty-assets that gets us plant genes and that's enabled through this, no?